tests/test_models.py

```diff
@@ -83,11 +83,15 @@ def test_credit_card(self):
     def test_device(self):
         id = 'b643d445-18b2-4b9d-bad4-c9c4366e402a'
         last_seen = '2016-06-08T14:16:38Z'
-        device = Device({'confidence': 99, 'id': id, 'last_seen': last_seen})
+        session_age = 3600
+        session_id = 'foobar'
+        device = Device({'confidence': 99, 'id': id, 'last_seen': last_seen, 'session_age': session_age, 'session_id': session_id })
 
         self.assertEqual(99, device.confidence)
         self.assertEqual(id, device.id)
         self.assertEqual(last_seen, device.last_seen)
+        self.assertEqual(session_age, device.session_age)
+        self.assertEqual(session_id, device.session_id)
```

Comment:
This will need to be replaced with tests in `test_validation.py`. Also, add the new fields to https://github.com/maxmind/minfraud-api-python/tree/9b15e69971bd27b9a5dc4282df78de5147ab0504/tests/data.